.travis.yml

```diff
@@ -24,6 +24,26 @@ jobs:
     after_success:
     - CODECOV_NAME=coverage.out bash <(curl -s https://codecov.io/bash)
 
+  # Build the Contour container image with the local Docker. Create a kind
+  # cluster, push the image we built into it, and verify that both Contour
+  # and Envoy come up. This is an initial placeholder as we experiment with
+  # different approaches to integration testing.
+  - name: integration
+    install:
+    - mkdir -p $HOME/bin
+    - ./hack/travis/install-kubernetes-toolchain.sh
+    - make container IMAGE=docker.io/projectcontour/contour VERSION=ci
+    - docker tag docker.io/projectcontour/contour:ci docker.io/projectcontour/contour:master
+    - docker tag docker.io/projectcontour/contour:ci docker.io/projectcontour/contour:latest
+    script:
+    - $HOME/bin/kind create cluster
+    - $HOME/bin/kind load docker-image docker.io/projectcontour/contour:master
+    - $HOME/bin/kind load docker-image docker.io/projectcontour/contour:latest
+    - $HOME/bin/kubectl apply -f examples/render/contour.yaml
```

Comment:
You need to wait for the kind cluster to be ready before this step. There's a wait flag now in kind to do this (` --wait duration`)

Reply:
Added.